packages/qdrant-loader/tests/.env.test.template

```
@@ -36,4 +36,10 @@ JIRA_EMAIL=your_jira_email_here
 # For Jira Data Center/Server
 # Personal Access Token (alternative to token/email for Data Center)
 # Generate at: Jira → Profile → Settings → Personal Access Tokens
-JIRA_PAT=your_jira_personal_access_token_here
\ No newline at end of file
+JIRA_PAT=your_jira_personal_access_token_here
+
+# Graph Database Configuration
+GRAPH_HOST=localhost
+GRAPH_PORT=6379
+GRAPH_NAME=test_graph
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GRAPH_NAME key should go before the GRAPH_PORT key

(UnorderedKey)